RESUME/api_views.py

```
@@ -1,12 +1,15 @@
 import logging
 from django.db.models import QuerySet
+from drf_spectacular.types import OpenApiTypes
 from drf_spectacular.utils import extend_schema, OpenApiParameter
-from rest_framework import status
+from rest_framework import status, viewsets
 from rest_framework.decorators import api_view
```

Ruff (0.8.2)

`rest_framework.decorators.api_view` imported but unused

Remove unused import: `rest_framework.decorators.api_view`

(F401)